api/src/controllers/auth.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file This module contains handlers for the "auth" route.
+ * @module auth
+ */
+const Team = require('../models/team.model');
+
+const bcrypt = require('bcrypt');
+
+const jwt = require('jsonwebtoken');
+
+const { fillErrorObject } = require('../middleware/error');
+
+const { aceessTokenExpiry, refreshTokenExpiry, accessTokenCookieExpiry, refreshTokenCookieExpiry } = require('../config/tokenExpiry');
+
+/**
+ * Handle login request from /team/login
+ * @param {*} req request object, containing team email and password in the body as JSON
+ * @param {*} res response object, the found team object with properties of teamId, email, teamName and orgName
+ * @returns 200: the team was found
+ * @returns 400: team is not found
+ */
+ async function login(req, res, next) {
+    const foundTeam = await Team.findOne({ email: req.body.email })
+    if (!foundTeam) { // team not found 
+      next(
+        fillErrorObject(400, 'Authenication error', [
+          'Incorrect email/password',
+        ])
+      );
+    } 
+
+    if (await bcrypt.compare(req.body.password, foundTeam.password)){
+      const jwtPayload = {
+        _id: foundTeam._id,
+        email: foundTeam.email,
+        teamName: foundTeam.teamName,
+        orgName: foundTeam.orgName
+      }
+
+      const accessToken = jwt.sign({ team: jwtPayload }, process.env.JWT_SECRET_ACCESS_TOKEN || "JWT_SECRET_ACCESS_TOKEN", {
```

Comment:
Out of curiosity, do you actually have a JWT_SECRET_ACCESS_TOKEN that you're using or is it defaulting to the string?

Reply:
No, haven't created it yet, I am just using the default string here.